test/basic.test.js

```diff
@@ -8,6 +9,81 @@ import { readFileSync } from 'fs'
 const fileUrl = new URL('../package.json', import.meta.url)
 const packageJSON = JSON.parse(readFileSync(fileUrl))
 
+test('"/federation-schema" returns a 404 error if fastify is intantiated without the gateway, and logs an error message', async t => {
```

Comment:
still missing the `s` -> inStantiated

Reply:
Argh, sorry about that. 

Fixed (and I've installed a spell checker too!)